jumpscale/sals/nettools/__init__.py

```diff
@@ -95,7 +95,7 @@ def wait_connection_test(ipaddr: str, port: int, timeout=5):
             return True
 
 
-def wait_http_test(url: str, timeout: int = 60) -> bool:
+def wait_http_test(url: str, timeout: int = 60, verify: bool = True) -> bool:
```

Comment:
Please update the usage of verify param in the docstring of the functionn

Reply:
done